### ISSUE TITLE
Communicate context to community

### ISSUE THREAD

**Issue by mrocklin** [numfocus/elections#5]
We're going to start taking actions (like #4 ) and we probably need to make a place where people can go to learn about what's going on, the objectives, the process, etc..

There are a few ways to do this:

- A blogpost somewhere
- The README of this repo
- A page on numfocus.org

Probably any one of these is fine.  We should then publish that thing, and then also link back to it in everything that we send out.  

@pavithraes I think that you're thinking about this right now, so I'm assigning you (but feel free to push back if that's premature)

**Comment by mrocklin** [comment]
My preference (very happy to be overruled) is to just make the README of this repo pretty comprehensive.  I like the idea of being github-forward in the election.  We'd point people back to this repo often, and they'd naturally look at the issues to find out more context.  

There are probably some negative impacts of this choice though.  I'll list a few below to be comprehensive

- GitHub might be offputting to people unfamiliar with OSS
- Maybe it feels more official if context is on a NumFOCUS asset, like numfocus.org
- It's kinda weird / atypical

**Comment by tkoyama010** [comment]
> - GitHub might be offputting to people unfamiliar with OSS
> - Maybe it feels more official if context is on a NumFOCUS asset, like numfocus.org
> - GitHub might be offputting to people unfamiliar with OSS

+1 Making README comprehensive. It would be nice to have a mechanism to deploy the README to numfocus.org, etc.

**Comment by ivanov** [comment]
I'm also on board with being github-forward / github-centric, I think the downside Matt listed are minor, given that NumFOCUS exists to serve open source projects, this repo is under the NF github organization, and being atypical is par for the course given the large ATX contingent, historical connection, and a certain duty to  "Keep Austin Weird" :)

**Comment by mrocklin** [comment]
@lagru I think that you're taking a first pass at this.  If it's easier than a PR, you could also try a Google Doc.  I'll bet that the collaborative editing there could be helpful in sharing this burden a bit more easily than github.  (we would then transcribe the doc over to a PR once edits had settled down).   Just a thought in case it makes things easier.

**Comment by lagru** [comment]
Okay, I've created an outline in a Google doc and shared in on the NumFOCUS slack channel (for now). Let's turn it into a PR later like you suggest. :)

**Comment by mrocklin** [comment]
Thanks for starting this @lagru 

After reading through it here are some thoughts (sorry for being critical!): 

- I think it could use more motivational front matter.  Probably 500 words that summarize what's going on, why it's important, and what we want people to do about it
- Right now it starts off with 1000 words with very specific detail about board member responsibility. I think that if we start with this then people won't read the rest of the document and won't engage
- maybe the thing to do here is to have a brief overview in the beginning and then have sections with detail afterwards that serve as a kind of appendix.  We could even link to them from the top
- I suspect that it is the way it is because it's been written by committee so far, which is natural.  I think that one person needs to take what's there and lead with stronger opinions.  Maybe this is you @lagru , but it could be anyone (hopefully not me) if you're unavailable
- this sounds silly, but I'll bet that a small visual numfocus logo would help make it seem more professional

I think that we're close, there's good content there.  Someone needs to turn that content into something concise, informative, and engaging. 

@lagru do you agree?  If so, is this you?

**Comment by lagru** [comment]
Hey @mrocklin, I didn't have resources last week to work on this beyond the initial outline. So your critique and suggestions are completely justified! I will try to look into this again this week, but I can't make any promises.

**Comment by lagru** [comment]
And considering the planned time line, I'd gladly welcome someone else taking the lead with this who has more time on hand right now.

**Comment by ivanov** [comment]
In terms of relating context to the community, I have [written up this post](https://pirsquared.org/blog/numfocus-concerns.html) which includes some concerns from the community, information about past elections, and the experiences of previous board members. I was mentally blocked on trying to push further on this committee without making these past experiences public for the benefit of potential candidates and voters.

**Comment by tkoyama010** [comment]
> In terms of relating context to the community, I have [written up this post](https://pirsquared.org/blog/numfocus-concerns.html) which includes some concerns from the community, information about past elections, and the experiences of previous board members. I was mentally blocked on trying to push further on this committee without making these past experiences public for the benefit of potential candidates and voters.

Thank you! I am a complete newbie to NumFOCUS so that this article would be helpful.

**Comment by aterrel** [comment]
I wrote some thoughts on this question for the election committee's consideration: https://docs.google.com/document/d/144-0_Fhnj2b1CORnT7tKN2xkT-Qn7qbikBs81-1Ltuo/edit?usp=sharing

**Comment by tkoyama010** [comment]
@all-contributors please add @ivanov for ideas and blog

**Comment by tkoyama010** [comment]
@all-contributors please add @aterrel for ideas
